src/game_map.h

```diff
@@ -142,7 +142,7 @@ namespace Game_Map {
 	 * @param self Character to move.
```

Comment:
force_through is missing in the doc comment

Reply:
Fixed.